redis_cloud/manifest.json

```diff
@@ -0,0 +1,72 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0b59b80e-db72-44a6-8c2b-67475d10ad71",
+  "app_id": "redis-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Redis Cloud Integration",
+    "title": "Redis Cloud",
+    "media": [
+      {
+        "media_type":  "image",
+        "caption": "closter overview displays",
+        "image_url": "images/datadog-overview-cluster.png"
+      },
+      {
+        "media_type":  "image",
+        "caption": "graph of node memory usage",
+        "image_url": "images/datadog-detail-node-memory.png"
```

Comment:
3 Images are required for the media carousel. This is what will display on your tile when a user clicks into it!